tests/network/sriov.go

```diff
@@ -605,6 +648,11 @@ func pciAddressExistsInGuest(vmi *v1.VirtualMachineInstance, pciAddress string)
 	return console.RunCommand(vmi, command, 15*time.Second)
 }
 
+func pciAddressExistsInGuestInterface(vmi *v1.VirtualMachineInstance, pciAddress, interfaceName string) error {
+	command := fmt.Sprintf("grep -q %s /sys/class/net/%s/device/uevent\n", pciAddress, interfaceName)
```

Comment:
nit: I would change the token to `PCI_SLOT_NAME=%s` to improve visibility, not a must though.

Reply:
you mean this?
```suggestion
	command := fmt.Sprintf("grep -q %s /sys/class/net/%s/device/uevent\n", pciSlotName, interfaceName)
```

Reply:
I think he means that given data as follows:
```
$ cat /sys/class/net/wlp0s20f3/device/uevent 
DRIVER=iwlwifi
PCI_CLASS=28000
PCI_ID=8086:06F0
PCI_SUBSYS_ID=8086:0070
PCI_SLOT_NAME=0000:00:14.3
MODALIAS=pci:v00008086d000006F0sv00008086sd00000070bc02sc80i00
```
To grep `PCI_SLOT_NAME=0000:00:14.3`.